currency/templatetags/currency_extras.py

```diff
@@ -0,0 +1,10 @@
+from django import template
+from django.contrib.auth.models import Group
+
+register = template.Library()
+
+
+@register.filter(name='is_group')
+def is_group(user, group_name):
```

Comment:
What `templatetags` mean? Why not create a file called `utils.py` in `currency` folder and keep this method there?

Reply:
It's a conventional way to realize some custom template tags
https://docs.djangoproject.com/en/3.2/howto/custom-template-tags/